helm_deploy/hmpps-nomis-visits-mapping-service/values.yaml

```diff
@@ -13,6 +13,7 @@ generic-service:
     enabled: true
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-nomis-visits-mapping-service-cert
+    contextColour: green
```

Comment:
We don't have that on some of the other services, what does it do?

Reply:
it is required for all CP deployments, `green` is `live` and `blue` is `live-1`.  The default is `blue` in the orb so needs overriding